src/set_theory/surreal/basic.lean

```diff
@@ -82,6 +82,21 @@ theorem numeric_rec {C : pgame → Prop}
 | ⟨l, r, L, R⟩ ⟨h, hl, hr⟩ :=
   H _ _ _ _ h hl hr (λ i, numeric_rec _ (hl i)) (λ i, numeric_rec _ (hr i))
 
+/-- Relabellings preserve being numeric. -/
+theorem numeric_congr {x y : pgame} (ox : numeric x) (r : relabelling x y) : numeric y :=
```

Comment:
This probably could benefit from dot notation; either as `relabelling.numeric_congr` or `numeric.relabelling_congr` or similar.

Also; can you provide this as an `iff` too?